src/actions/categoriasActions.js

```diff
@@ -16,31 +15,28 @@ export const getCategoryById = async (id) => {
   }
 };
 
-export const getCategories = async () => {
-  try {
-    const res = await authAxios.get("/category");
-
-    if (res.data.status === STATUS_OK) {
-      return res.data.body.data;
-    } else {
-      return false;
-    }
-  } catch (error) {
-    response.error(error);
-    return false;
-  }
-};
+const categories = [
+  "African Art",
+  "American Painting and Sculpture",
+  "Art of the Americas",
+  "Chinese Art",
+  "Contemporary Art",
+  "Decorative Art and Design",
+  "Drawings",
+  "Egyptian and Ancient Near Eastern Art",
+  "European Painting and Sculpture",
+  "Greek and Roman Art",
+  "Indian and South East Asian Art",
+  "Islamic Art",
+  "Japanese Art",
+  "Korean Art",
+  "Medieval Art",
+  "Modern European Painting and Sculpture",
+  "Oceania",
+  "Performing Arts, Music, & Film",
+  "Photography",
+  "Prints",
+  "Textiles",
+];
```

Comment:
What do you think about exporting this "categories" array in a new JSON file? to easy handle